examples/README-CN.md

```diff
@@ -0,0 +1,43 @@
+# 示例
+
+这些示例默认 `kustomize` 在您的 `$PATH` 中。
+
+这些示例通过了 [pre-commit](../bin/pre-commit.sh) 测试，并且应该与 HEAD 一起使用。
+
+<!-- @installkustomize @test -->
+```
+go get sigs.k8s.io/kustomize
+```
+
+ * [hello world](helloWorld/README.md) - 部署多个配置不同的 Hello World 服务。
+
+ * [last mile helm](chart.md) - 对 helm chart 进行 last mile 修改。
+   
+ * [LDAP](ldap/README.md) - 部署多个配置不同的 LDAP 服务。
+
+ * [mySql](mySql/README.md) - 从头开始创建一个 MySQL 的生产配置。
+
+ * [springboot](springboot/README.md) - 从头开始创建一个 Spring Boot 项目的生产配置。
+
+ * [combineConfigs](combineConfigs.md) -
+   混合来自不同用户的配置数据（例如来自 devops/SRE 和 developers）。
+   
+ * [configGenerations](configGeneration.md) - 当 ConfigMapGenerator 修改时进行滚动更新。
+
+ * [secret generation](kvSourceGoPlugin.md) - 生成 Secret。
+ 
+ * [generatorOptions](generatorOptions.md) -修改所有 ConfigMapGenerator 和 SecretGenerator 的行为。
+
+ * [breakfast](breakfast.md) - 给 Alice 和 Bob 定制一顿早餐 :)
```

Comment:
Yes, this example is serious :)
It demonstrates that Kustomize works with any types as long as those YAML follows kubernetes styles, e.g. containing apiVersion, kind, metadata.name.

Reply:
If breakfast isn't appropriate, we could refactor it into lunch.

Reply:
> If breakfast isn't appropriate, we could refactor it into lunch.

Breakfast is great. I love breakfast..._(´ཀ`」 ∠)_...Congratulations Coffee and  become k8s resource.